fiftyone/factory/repos/execution_store.py

```diff
@@ -36,7 +44,10 @@ def __init__(self, dataset_id: Optional[ObjectId] = None):
 
     @abstractmethod
     def create_store(
-        self, store_name: str, metadata: Optional[Dict[str, Any]] = None
+        self,
+        store_name: str,
+        metadata: Optional[Dict[str, Any]] = None,
+        policy: str = "persist",
```

Comment:
So, `policy` applies for both keys and for stores?

Reply:
Yes